backend/src/packages/users/user.service.ts

```diff
@@ -1,4 +1,6 @@
 import { type IService } from '~/libs/interfaces/interfaces.js';
+import { type IConfig } from '~/libs/packages/config/config.js';
+import { type IEncrypt } from '~/libs/packages/encryption/libs/interfaces/encrypt.interface.js';
```

Comment:
Name folder the same way as it's file, and add needed interface to reexport
```suggestion
import { type IEncrypt } from '~/libs/packages/encrypt/encrypt.js';
```